.github/workflows/release.yaml

```
@@ -100,26 +103,10 @@ jobs:
           fpm_args: '.'
           fpm_opts: "-t rpm -v ${{ env.VERSION }} -C ./build --depends snappy ${{ env.FPM_OPTS }}"
 
-      - name: Upload Rpm Package
-        uses: kittaakos/upload-artifact-as-is@v0.0.1
-        with:
-          path: ./*.rpm
-
-  release:
-    name: Release Linux Packages
-    needs: [ build-deb-packages-and-push-docker-image, build-rpm-packages ]
-    runs-on: ubuntu-18.04
-    steps:
-
-      - name: Download Linux Packages
-        uses: actions/download-artifact@v2
-
       - name: Release
         uses: softprops/action-gh-release@v1
         with:
           files: |
-            ./*.deb
             ./*.rpm
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

```suggestion
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}

```
we should keep one empty line 🤣

---

kvrocks.yaml did't have one empty line, change kvrocks.yaml to have one empty line too in this PR？

---

oh, my bad, i forget, it doesn't matter, to make this PR doesn't influence other things, we needn't